### DISCUSSION TITLE
How to handle a mutation that returns a value?

### DISCUSSION THREAD

**Discussion by dphinc** [shurcooL/graphql#86] [Q&A]
How would you code the following mutation that returns an ID upon creation:

```
mutation CreateTeamPipeline {
  teamPipelineCreate(input: {teamID: "...", pipelineID: "...", accessLevel: READ_ONLY}) {
    teamPipeline {
      id
    }
  }
}
```

**Comment by dmitshur** [comment]
You should define a variable with the corresponding structure that includes an `ID` field, and pass it to the `Mutate` method. The `ID` field in the variable you passed will be populated.

Have you looked at the [Mutations](https://github.com/shurcooL/graphql#mutations) section in the README?
